utils/swift_build_support/swift_build_support/products/swiftpm.py

```diff
@@ -42,7 +42,7 @@ def is_before_build_script_impl_product(cls):
     def should_build(self, host_target):
         return True
 
-    def run_bootstrap_script(self, action, host_target, additional_params=[]):
+    def run_bootstrap_script(self, action, host_target, additional_params=[], *, ignore_cross_compile_argument = False):
```

Comment:
I would suggest to rename the argument to state its purpose more clearly, e.g.:

```suggestion
    def run_bootstrap_script(self, action, host_target, additional_params=[], *, compile_only_for_running_host_architecture = False):
```

Reply:
Done